MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -603,13 +603,10 @@ + (void)videoChangeForPlayer:(nonnull NSString *)name withCustomerData:(nullable
     MUXSDKCustomerVideoData *videoData = [customerData customerVideoData];
     MUXSDKCustomData *customData = [customerData customData];
     
-    if (!(videoData || viewData || customData)) {
```

Comment:
Does removing this change anything?

Reply:
This would prevent the `dispatchViewEnd` and `dispatchVideoChange` calls from going out, so there'd be no video change. I don't think that makes sense to do from an API standpoint.

The customer doesn't necessarily have to change any of their customer data when they do `videochange`. They may choose to do so later on in the new view via `setCustomerData`, may only provide info that wouldn't need to change, or may choose not to add any additional info at all.

Reply:
Agree that the updated behavior makes more sense. We should probably call this out in the release notes so that anyone relying on how this works is aware of the change.

Reply:
(And thanks for the explainer!)